src/documents/serialisers.py

```diff
@@ -458,7 +460,11 @@ def create(self, validated_data):
         return instance
 
     def get_value(self, obj: CustomFieldInstance):
-        return obj.value
+        return (
+            obj.value
+            if (obj.field.data_type != CustomField.FieldDataType.DOCUMENTLINK)
+            else json.loads(obj.value)
```

Comment:
Since it looks like you are using the new field as a JSON field, why not use a `JSONField`?